fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/RewriteTreeTask.java

```diff
@@ -96,10 +96,6 @@ private void rewrite(OptExpression parent, int childIndex, OptExpression root) {
     }
 
     private boolean match(Pattern pattern, OptExpression root) {
-        if (!pattern.matchWithoutChild(root)) {
-            return false;
```

Comment:
could you explain the reason of deleting this?

Reply:
sorry, code merge error